From 77fd6a14ca3596904f08e25ddd3274b21588df80 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 31 May 2022 18:12:41 +0200
Subject: [PATCH 175/841] feat(stm32mp2): add BKPSRAM base address

Define STM32MP_BACKUP_RAM_BASE as the base address for BKPSRAM.
This is the same macro as STM32MP1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git b4f4bf57ba1d05eff5f0ec1c8833e85a00afbb4c feat(stm32mp2): add BKPSRAM base address]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I69d79f45b42899f1f0688d78a64f19c04aea448f
---
 plat/st/stm32mp2/stm32mp2_def.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index af06aded5..84d843b23 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -59,6 +59,7 @@
 #define STM32MP_SYSRAM_SIZE		U(0x00040000)
 #define SRAM1_BASE			U(0x0E040000)
 #define SRAM1_SIZE			U(0x00020000)
+#define STM32MP_BACKUP_RAM_BASE		U(0x42000000)
 
 #define STM32MP_NS_SYSRAM_SIZE		PAGE_SIZE
 #define STM32MP_NS_SYSRAM_BASE		(STM32MP_SYSRAM_BASE + \
-- 
2.47.2

